components/connector-service/internal/clientcontext/extractor.go

```diff
@@ -12,6 +12,7 @@ import (
 const (
 	MetadataURLFormat = "%s/%s/v1/metadata/services"
 	EventsURLFormat   = "%s/%s/v1/events"
+  EventsInfoURLFormat = "%s/%s/v1/events/subscribed"
```

Comment:
please consider not naming it `subscribed`. As discussed previously with @akgalwas, there could be scenarios where there is no Kyma Subscription but still the event needs to be consumed e.g. for data events which will be consumed by a data event processing framework.

Reply:
After discussion with team and PO, we decided to stay with `subscribed`. It is general term in case of events, not necessarily pointing to Kyma Subscriptions.